src/extra/gfx/gltf.cpp

```diff
@@ -16,13 +16,14 @@ struct GLTFShard {
   static inline Type PathInputType = CoreInfo::StringType;
   static inline Type ByteInputType = CoreInfo::BytesType;
   static inline Type TransformVarType = Type::VariableOf(CoreInfo::Float4x4Type);
+
   static SHTypesInfo inputTypes() { return CoreInfo::AnyTableType; }
   static SHTypesInfo outputTypes() { return Types::DrawableHierarchy; }
 
   static SHParametersInfo parameters() {
     static Parameters params{
-        {"Transform", SHCCSTR("The transform variable to use"), {TransformVarType}},
-        {"Path", SHCCSTR("The static path to load a model from during warmup"), {CoreInfo::StringType}},
+        {"Transform", SHCCSTR("The transform variable to use (Optional)"), {CoreInfo::NoneType, TransformVarType}},
+        {"Path", SHCCSTR("The static path to load a model from during warmup (Optional)"), {CoreInfo::NoneType, CoreInfo::StringType}},
```

Comment:
Can use `CoreInfo::StringOrNone`

In addition, because the string is optional, `setParam()` and `getParam()` should be adpated (see for example in `imgui.cpp` lines 794-884):
```cpp
  void setParam(int index, const SHVar &value) {
    switch (index) {
    case 0:
      _transformVar = value;
      break;
    case 1:
      if (value.valueType == None) {
        _staticModelPath .clear();
      } else {
        _staticModelPath = value.payload.stringValue;
      }
      break;
    }
  }
  SHVar getParam(int index) {
    switch (index) {
    case 0:
      return _transformVar;
    case 1:
      return _staticModelPath.size() == 0 ? Var::Empty : Var(_staticModelPath);
    default:
      return Var::Empty;
    }
  }
```

Reply:
Ok added that